source/guide_beehive.rst

```diff
@@ -0,0 +1,124 @@
+.. author:: momoaux <momoaux@koma666.de>
+
+.. tag:: web
+.. tag:: agent
+.. tag:: lang-go
+
+.. highlight:: console
+
+.. sidebar:: Logo
+
+  .. image:: _static/images/beehive.png
+      :align: center
+
+#####
+Beehive
+#####
+
+.. tag_list::
+
+Beehive is an event and agent system, which allows you to create your own
+agents that perform automated tasks triggered by events and filters. It is
+modular, flexible and really easy to extend for anyone. It has modules
+(we call them *Hives*), so it can interface with, talk to, or retrieve
+information from Twitter, Tumblr, Email, IRC, Jabber, RSS, Jenkins, Hue - to
+name just a few. Check out the full list of [available Hives](https://github.com/muesli/beehive/wiki/Available-Hives)
+in our Wiki.
+
+Connecting those modules with each other lets you create immensly useful agents.
+----
+
+.. note:: For this guide you should be familiar with the basic concepts of
+
+  * :manual:`Go <lang-go>`
+  * :manual:`Web Backends <web-backends>`
+  * :manual:`Supervisord <daemons-supervisord>`
+  * :manual:`Domains <web-domains>`
+
+License
+=======
+
+Beehive is released under the `AGPL-3.0 License`_.
```

Comment:
This link [does not seem to work](https://deploy-preview-925--uberlab.netlify.app/guide_beehive.html#license). Maybe because there is an extra ` ` in front of `AGPL` in the link list at the bottom?

Reply:
Hope i updated the links right. Can I check the preview without bothering you everytime when I update? could not find something here: https://github.com/Uberspace/lab/blob/main/STYLE.md